.maestro/app_tp/app_tp_onboarding.yaml

```diff
@@ -1,4 +1,7 @@
 appId: com.duckduckgo.mobile.android
+name: "SmokeTest: AppTP onboarding"
```

Comment:
nit: rename to Release Test to be consistent with the tag.

Reply:
Good spot